exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>typos: "which has 3 standard preprocessing steps"</p><p> </p><p>"by setting the respective model to None". Put "use a different(typo) model ..." in a different sentence, otherwise it sounds like it is another way to disable the extractor.</p><p> </p><p>instead of the "built-in one".</p><p> </p><p>The topic classification model will use the topics "urgent", and "not urgent".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>